MediaGalleryUi/Model/AssetIndexer.php

```diff
@@ -80,6 +80,7 @@ public function execute(\SplFileInfo $item): void
                     'path' => $this->getMediaDirectory()->getRelativePath($file),
                     'title' => $item->getBasename('.' . $item->getExtension()),
                     'created_at' => (new \DateTime())->setTimestamp($item->getCTime())->format('Y-m-d H:i:s'),
+                    'updated_at' => (new \DateTime())->setTimestamp($item->getMTime())->format('Y-m-d H:i:s'),
```

Comment:
Sorry, I thought that we need to save the timestamp for the image file. Cause it will give improper result to user if we save the timestamp automatically in the database. Correct me if I'm wrong?

Reply:
Yeah, you are also right. I just think that the image data in the database is supposed to be updated more frequently than image file (that is unlikely to be updated at all), so knowing modification date of the asset entity should be more informative to admin user than the modification date of the image file.

@tkacheva correct me if I'm wrong.